### PR TITLE
Harvester shell integration - remove LocaleSelector when is Harvester standalone UI

### PR DESCRIPTION
**Pull request by torchiaf**
### Summary
Contributes to https://github.com/rancher/dashboard/issues/12347

### Checklist
- [x] The PR is linked to an issue and the linked issue has a Milestone, or no issue is needed
- [x] The PR has a Milestone 
- [x] The PR template has been filled out
- [x] The PR has been self reviewed 
- [x] The PR has a reviewer assigned
- [x] The PR has automated tests or clear instructions for manual tests and the linked issue has appropriate QA labels, or tests are not needed
- [x] The PR has reviewed with UX and tested in light and dark mode, or there are no UX changes

### CODE REVIEW

File: shell/pages/auth/login.vue [hunk 1]
```diff
@@ -64,7 +64,12 @@ export default {
   },
 
   computed: {
-    ...mapGetters({ t: 'i18n/t' }),
+    ...mapGetters(['isSingleProduct']),
+    ...mapGetters({ t: 'i18n/t', hasMultipleLocales: 'i18n/hasMultipleLocales' }),
+
+    isHarvester() {
+      return this.isSingleProduct?.productName === 'harvester';
```
Comment (a110605) [inline shell/pages/auth/login.vue:71]: nit: replace with  ?
```
import { HARVESTER_NAME as HARVESTER } from '@shell/config/features';
```

File: shell/pages/auth/login.vue [hunk 2]
```diff
@@ -64,7 +64,12 @@ export default {
   },
 
   computed: {
-    ...mapGetters({ t: 'i18n/t' }),
+    ...mapGetters(['isSingleProduct']),
+    ...mapGetters({ t: 'i18n/t', hasMultipleLocales: 'i18n/hasMultipleLocales' }),
```
Comment (a110605) [inline shell/pages/auth/login.vue:68]: We don't want to show LocalSelector when harvester is standalone mode, right?

How about using `isStandaloneHarvester`?
```
mapGetters(['isStandaloneHarvester']),
```

**Comment by a110605** [comment]
Please fix lint error

File: shell/pages/auth/login.vue (hunk 2 above)
Comment (torchiaf) [reply shell/pages/auth/login.vue:68]: Fixed

**Review by a110605 — APPROVED** [review]
LGTM

**Pull request merged**